br/backup-and-restore-design.md

```diff
@@ -29,15 +29,50 @@ BR can generate the following types of backup files:
 
 ### Naming format of SST files
 
-SST files are named in the format of `storeID_regionID_regionEpoch_keyHash_cf`. The fields in the format are explained as follows:
+SST files are named in the format of `storeID_regionID_regionEpoch_keyHash_timestamp_cf`. The fields in the format are explained as follows:
 
 - `storeID` is the TiKV node ID.
 - `regionID` is the Region ID.
 - `regionEpoch` is the version number of a Region.
 - `keyHash` is the Hash (sha256) value of the startKey of a range, which ensures the uniqueness of a key.
+- `timestamp` is the Unix timestamp of an SST file when it is generated at TiKV.
 - `cf` indicates the Column Family of RocksDB (`default` or `write` by default).
 
+When data is backed up to Amazon S3 or a network disk, the SST files are named in the format of `regionID_regionEpoch_keyHash_timestamp_cf`.
+
 ### Storage format of SST files
 
 - For details about the storage format of SST files, see [Rocksdb BlockBasedTable Format](https://github.com/facebook/rocksdb/wiki/Rocksdb-BlockBasedTable-Format).
 - For details about the encoding format of backup data in SST files, see [Mapping of table data to Key-Value](/tidb-computing.md#mapping-of-table-data-to-key-value).
+
+### Backup file layout
```

Comment:
```suggestion
### Backup file structure
```
I think [file structure](https://www.google.com/search?q=file+structure&newwindow=1&source=lnms&tbm=isch&sa=X&ved=2ahUKEwiBw9qCjfD5AhXjGTQIHfx9AfgQ_AUoAXoECAEQAw&biw=960&bih=944&dpr=2) is more accurate in this case.